### DISCUSSION TITLE
Security Trim resutls

### DISCUSSION THREAD

**Discussion by edgarhernandez2025** [microsoft-search/pnp-modern-search#3223] [Q&A]
**Version used**
4.9.2.0

**Describe the bug**
I am getting results when a user doesn't have permission in the file. 

**To Reproduce**
Detailed steps to reproduce the behavior:
1. Create a document library A, add some document and provide access just to UserA
2. Create a document library B, add some document and provide access just to UserB
3. Index the site
4. Search for document using the UserA
5. In the search results the documents in the Library B are displayed (The user doesn't have access to the Library B)

**Expected behavior**
The search result will not display the document in the Library B

**Desktop (please complete the following information):**
 - Browser Chrome, Edge

**Comment by kasperbolarsen** [comment]
Please check if the same apply in the classic search center and/Or in Microsoft Search 

Most likely the reason is that the search index hasn't been updated since you broke the permissions on the libraries. We sometimes see cases where content isn't crawled for hours or even days!!
If you are familiar with PowerShell you can query the crawl log to check.
